File: Linguagem C Exercicios/C-Condicional/TROCO VERIFICADO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double troco,recebido,precoUn,precoTotal = 0;
    int qnt = 0;

    printf("Digite o valor unitario da peca: R$ ");
    scanf("%lf",&precoUn);
    printf("Digite a quantidade comprada: ");
    scanf("%d",&qnt);
    printf("Digite o valor recebido: R$ ");
    scanf("%lf",&recebido);

    precoTotal = precoUn * qnt;

    if (recebido >= precoTotal){
        troco = recebido - precoTotal;
        printf("TROCO = %.2lf",troco);
    }else {
        troco = precoTotal - recebido;
        printf("DINHEIRO INSUFICIENTE. AINDA FALTAM %.2lf REAIS",troco);
    };


    return 0;
}

File: Linguagem Java/Exercicios/TrocoVerificado.java
import java.util.Locale;
import java.util.Scanner;

public class TrocoVerificado {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double precoUn,dinheiroRecebido,precoTotal,troco;
		int qnt;
		
		System.out.print("Preco unitario do produto: ");
		precoUn = sc.nextDouble();
		System.out.print("Quantidade comprada: ");
		qnt = sc.nextInt();
		System.out.print("Dinheiro recebido: ");
		dinheiroRecebido = sc.nextDouble();
		
		precoTotal = qnt * precoUn;
		
		if (dinheiroRecebido > precoTotal) {
			troco = dinheiroRecebido - precoTotal;
			System.out.println("TROCO = "+String.format("%.2f", troco));
		}else {
			troco = precoTotal - dinheiroRecebido;
			System.out.println("DINHEIRO INSUFICIENTE, FALTAM "+String.format("%.2f", troco));			
		};
		
		sc.close();	

	}

}
